packages/aws-cdk-lib/aws-ecs/lib/base/service-managed-volume.ts

```diff
@@ -179,6 +183,10 @@ export enum EbsPropagatedTagSource {
    * TASK_DEFINITION
    */
   TASK_DEFINITION = 'TASK_DEFINITION',
+  /**
+   * NONE
+   */
+  NONE = 'NONE',
```

Comment:
as per [the documentation ](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-service.html#cfn-ecs-service-propagatetags) .. None is not allowed value.